src/routes.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Route } from 'react-router';
+import Epics from './pages/Epics/Epics';
+import Home from './pages/Home/Home';
+import { Switch } from "react-router-dom";
+
+// Configuration of routes
+// TODO - add to list of routes
+const ROUTES = [
```

Comment:
Why are we using this instead of the browser router?

Reply:
BrowserRouter is wrapped around index.js. This file is a route config that will allow us to tack on routes in a much cleaner and scalable way, especially if routes have subroutes.